fdbclient/admin_client.go

```diff
@@ -151,75 +162,90 @@ func (command cliCommand) getTimeout() time.Duration {
 	return DefaultCLITimeout
 }
 
+// getVersion returns the versions defined in the command or if not present returns the running version of the
+// cluster.
+func (command cliCommand) getVersion(cluster *fdbv1beta2.FoundationDBCluster) string {
```

Comment:
This looks like we return two different answers out of a single method?

Reply:
Not really. The logic is the follow:

- If the command has a version defined use that version (e.g. backup commands).
- Otherwise use the running version of the cluster (all calls in the cluster controller).